templates/base.html.twig

```diff
@@ -23,43 +23,85 @@
 
         <nav class="navbar navbar-expand-lg navbar-dark" id="mainNav">
             <div class="container">
-                <a class="navbar-brand js-scroll-trigger" href="#page-top">Start Bootstrap</a>
+                <a class="navbar-brand js-scroll-trigger" href="/">Travel Mate</a>
                 <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
                     Menu
                     <i class="fa fa-bars"></i>
                 </button>
                 <div class="collapse navbar-collapse" id="navbarResponsive">
                     <ul class="navbar-nav text-uppercase ml-auto">
+
+
+                        {% if app.user.roles is defined %}
```

Comment:
jei nori patikrinti ar useris prisijunges, geriau rasyk tiesiog `{% if app.user %}`

https://stackoverflow.com/questions/9660764/symfony-2-how-do-i-check-if-a-user-is-not-logged-in-inside-a-template/9660881#9660881